js/logo.js

```diff
@@ -1219,6 +1220,9 @@ function Logo(pitchtimematrix, pitchdrummatrix, rhythmruler,
                 if (typeof(args[0]) === 'string') {
                     logo.errorMsg(NANERRORMSG, blk);
                     logo.stopTurtle = true;
+                } else if (args[0] < 0) {
```

Comment:
Maybe less than or equal to zero? And maybe not a fatal error, but just display the error message and  skip over the repeat stack?